### ISSUE TITLE
navigator.clipboard.writetext in android does not work, it throws Document is not focused.

### ISSUE THREAD

**Issue by linxianxi** [react-native-webview/react-native-webview#2480]
in my webview page, it has a button, click the button will use navigator.clipboard.writetext to copy the text, but does not work

**Comment by matheus-hora-guiavet** [comment]
I had the same issue recently. Use https://github.com/harisbaig100/react-native-clipboard. You can send a message from web to react native webview to get info and copy it on native side with this lib.

**Comment by sibelius** [comment]
how to use the https://github.com/harisbaig100/react-native-clipboard with this package?

**Comment by Bernardo1411** [comment]
> how to use the https://github.com/harisbaig100/react-native-clipboard with this package?

Based on what @matheus-hora-guiavet said, this is how I solved it:

On the React web app that I'm using on my webview, I sent the content to the React Native app:

```
onClick={() => { 
  window.ReactNativeWebView.postMessage("content you want to copy to clipboard"))
  }
}
```

`window.ReactNativeWebView.postMessage` basically sends any content from webview to React Native.

On my RN app, I installed [@react-native-community/react-native-clipboard](https://www.npmjs.com/package/@react-native-community/react-native-clipboard) and then on onMessage prop on RN webview, I called the method `Clipboard.setString`:

```
import Clipboard from '@react-native-community/react-native-clipboard';

function App() {
  <webview
    onMessage={(event) => {
      Clipboard.setString(event.nativeEvent.data);
      }
    }
  />
}
```

`event.nativeEvent.data` is the content sent from webview to RN.

This is a pretty simple example, but I hope it helps.
